examples/reghd.py

```diff
@@ -0,0 +1,111 @@
+# The following two lines are only needed because of this repository organization
+import sys, os
+
+sys.path.insert(1, os.path.realpath(os.path.pardir))
+
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils.data as data
+
+import torchmetrics
+from tqdm import tqdm
+
+from torchhd import functional
+from torchhd import embeddings
+from torchhd.datasets.airfoil_self_noise import AirfoilSelfNoise
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+print("Using {} device".format(device))
+
+DIMENSIONS = 10000  # number of hypervector dimensions
+NUM_FEATURES = 5  # number of features in dataset
+
+# Import dataset
+ds = AirfoilSelfNoise("../data", download=True)
+
+# Get statistics for normalization of input features
+STD_DEVS = ds.data.std(0)
+MEANS = ds.data.mean(0)
+
+TARGET_STD = ds.targets.std(0)
+TARGET_MEAN = ds.targets.mean(0)
+
+
+def transform(x):
+    x = x - MEANS
+    x = x / STD_DEVS
+    return x
+
+
+def target_transform(x):
+    x = x - TARGET_MEAN
+    x = x / TARGET_STD
+    return x
+
+
+ds = AirfoilSelfNoise(os.getcwd(), transform=transform, download=False)
```

Comment:
`ds` is declared twice why? If this is because you need to set the transforms you can also do:

```python
ds.tranform = transform
ds.target_transform = target_transform
```